Solutions/Threat Intelligence/Analytic Rules/IPEntity_DuoSecurity.yaml

```diff
@@ -10,6 +10,9 @@ requiredDataConnectors:
   - connectorId: ThreatIntelligenceTaxii
     dataTypes:
       - ThreatIntelligenceIndicator
+  - connectorId: CiscoDuoSecurity
+    dataTypes:
+      - CiscoDuo
 queryFrequency: 1h
 queryPeriod: 14d
 triggerOperator: gt
```

Comment:
Query is missing 
| summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId

Reply:
Added done.